frontend/src/actions/index.js

```diff
@@ -10,6 +10,7 @@ import {
   fetchAllCandidates,
   addFilter,
   removeFilter,
+  resetFilter,
```

Comment:
not within the scope of this pr but a general comment - why don't we just `export {...} from "./actionCreators.js"` instead of repeating everything twice?

Reply:
Haven't taken a look into this but I can loop it in to my cleanup commit if it's not too much work

Reply:
it is done